tree/dataframe/inc/ROOT/RNTupleDS.hxx

```diff
@@ -58,6 +58,14 @@ class RNTupleDS final : public ROOT::RDF::RDataSource {
    unsigned fNSlots = 0;
    bool fHasSeenAllRanges = false;
 
+   /// @brief Holds useful information about fields added to the RNTupleDS
+   struct RFieldInfo {
+      std::size_t fFieldId;
```

Comment:
Shouldn't this be an RNTuple `DescriptorId_t` instead?
```suggestion
      DescriptorId_t fFieldId;
```

Reply:
Yes, implemented in the latest commits